packages/commonwealth/client/scripts/views/pages/CommunityManagement/Contests/ManageContest/steps/DetailsFormStep/DetailsFormStep.tsx

```diff
@@ -54,13 +61,24 @@ const DetailsFormStep = ({
   onSetContestFormData,
 }: DetailsFormStepProps) => {
   const navigate = useCommonNavigate();
+  const farcasterContestEnabled = useFlag('farcasterContest');
+  const [searchParams] = useSearchParams();
+  const contestType = searchParams.get('type');
+  const isFarcasterContest = contestType === ContestType.Farcaster;
```

Comment:
should we check for lettercase?

Reply:
not necessary, we own the enum so there is no chance for this mistake (unless you do it deliberately)